### PR TITLE
Merge  v0.4.3

### PR DESCRIPTION
**Pull request by mprevide**

### CODE REVIEW

File: tests/docker-compose.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ services:
         max-size: 100m
 
   data-broker:
-    image: dojot/data-broker:latest
+    image: dojot/data-broker:v0.4.2
```
Comment (eduardogmisiuk) [inline tests/docker-compose.yml:64]: Shouldn't it be 0.4.3?

Comment (mprevide) [reply tests/docker-compose.yml:64]: Version v0.4.3 does not yet exist.
This is used only by automated integration tests.

**Review by eduardogmisiuk — APPROVED** [review]
LGTM!

**Review by thiagodpf — APPROVED** [review]
:+1:

**Pull request merged**